dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -1460,7 +1462,40 @@ class CompileTest {
       case DeviceOperatingSystem.fuchsia:
         throw Exception('Unsupported option for Fuchsia devices');
       case DeviceOperatingSystem.macos:
-        throw Exception('Unsupported option for macOS devices');
+        unawaited(stderr.flush());
+        options.insert(0, 'macos');
+        options.add('--tree-shake-icons');
+        options.add('--split-debug-info=infos/');
+        watch.start();
+        await flutter('build', options: options);
+        watch.stop();
+        final String buildDirectoryPath = path.join(
+          cwd,
+          'build',
+          'macos',
+          'Build',
+          'Products',
+          'Release',
+        );
+        final String? appBundlePath =
+            _findIosAppInBuildDirectory(buildDirectoryPath);
```

Comment:
Rename this
```suggestion
            _findAppInBuildDirectory(buildDirectoryPath);
```

Reply:
Before I make the change - I want to provide some additional context so that we are on the same page.

This is the logic in the iOS branch of the switch case for `_compileTest`.
https://github.com/flutter/flutter/blob/a6c2e711e455bed6948f4f57620840e71c3d9eba/dev/devicelab/lib/tasks/perf_tests.dart#L1411-L1416

I copied it over to the macOS branch of the switch case, but I noticed [`_findIosAppInBuildDirectory`](https://github.com/flutter/flutter/blob/a6c2e711e455bed6948f4f57620840e71c3d9eba/dev/devicelab/lib/tasks/perf_tests.dart#L1946-L1953) was being used elsewhere to grab the `*.app` directory for iOS.

https://github.com/flutter/flutter/blob/a6c2e711e455bed6948f4f57620840e71c3d9eba/dev/devicelab/lib/tasks/perf_tests.dart#L1946-L1953

If I use the function call, I will rename to `_findAppInBuildDirectory(buildDirectoryPath)` since it will be used by`iOS` and `macOS`. Let me know what you think about that.

While I'm at it, should I refactor this: https://github.com/flutter/flutter/blob/a6c2e711e455bed6948f4f57620840e71c3d9eba/dev/devicelab/lib/tasks/perf_tests.dart#L1411-L1416

Reply:
@jmagman comment is outdated due to the following change
https://github.com/flutter/flutter/blob/d70df45883b810910e8b90a9e5aec1f750869b7b/dev/devicelab/lib/tasks/perf_tests.dart#L1960-L1968